crates/ruma-events/src/room/message.rs

```diff
@@ -152,6 +155,133 @@ impl RoomMessageEventContent {
         forward_thread: ForwardThread,
         add_mentions: AddMentions,
     ) -> Self {
+        self = self.make_reply_fallback(original_message.into());
```

Comment:
Maybe use a new binding here instead of having a `mut self`?

Reply:
Do you mean that the method should take a `self` instead of a `mut self`?

Reply:
Yeah; not like it matters for callers, but IMHO fewer `mut` bindings is generally better.

Reply:
To be completely clear, I mean

```suggestion
        let with_reply_fallback = self.make_reply_fallback(original_message.into());
```

and using `with_reply_fallback` instead of `self` below.